dangerzone/conversion/common.py

```diff
@@ -19,8 +19,27 @@
 
 def running_on_qubes() -> bool:
     # https://www.qubes-os.org/faq/#what-is-the-canonical-way-to-detect-qubes-vm
-    if os.environ.get("DZ_USE_CONTAINERS", "0") == "0":
-        return os.path.exists("/usr/share/qubes/marker-vm")
+    return os.path.exists("/usr/share/qubes/marker-vm")
+
+
+def qubes_native_conversion() -> bool:
```

Comment:
Τhe purpose of this function is to check if the conversion should run in a disposable qube. With that in mind, it probably won't run within the conversion sandbox. So I'd propose moving it to the Qubes isolation provider module.

Reply:
Done in [6ec1b0b](https://github.com/freedomofpress/dangerzone/pull/464/commits/6ec1b0b15e6824c34a039540d4e9331773c5253c)

Reply:
Great, thanks a lot for this change. One more thing, since we now moved this function under `dangerzone/isolation_provider/qubes.py`, I believe we can remove this logic:

```
# In Qubes we can safely access parent modules
from ..util import get_resource_path
```

Reply:
Done in [6ec1b0b](https://github.com/freedomofpress/dangerzone/pull/464/commits/6ec1b0b15e6824c34a039540d4e9331773c5253c) and [e6020da](https://github.com/freedomofpress/dangerzone/pull/464/commits/e6020da6adc507ec9d25c1cb15d98c0b10c95f23).